YvesIraguha/personalFinanceApi
# Filter Records Between Provided Start Date and End Date

#### What does this PR do?
- retrieve records between particular dates
#### Description of Task to be completed?
- Only return records between provided particular dates
#### How should this be manually tested?
- clone the repo and check out the branch `fix-pictureUrl-investment`
- Updated your `.env` file
- start the server by running `npm run dev`
- You can now test the implementation by issuing the start date and end date on `getAllInvestments` query 
#### Any background context you want to provide?
- When a user provides a start date, and end date, records retrieved fall between those dates. However, when a user provides nothing, all records are returned.
